artman/config/proto/user_config.proto

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package googleapis.artman;
+
+// Artman user config is used to store user settings. All configuration are
+// optional unless being explicitly marked required. Having some configuration
+// will simplify the CLI invocation (e.g. github config), or change the
+// execution behavior (e.g. local.toolkit).
```

Comment:
> change the execution behavior (e.g. local.toolkit)

What does this mean? if `local.toolkit` is specified, it doesn't cause artman always to run in local mode, right?

Reply:
Added clarification.